src/nvme/ioctl.h

```diff
@@ -5099,15 +5125,16 @@ static inline int nvme_copy(nvme_link_t l, __u32 nsid, __u64 sdlba, __u16 nr, __
  */
 static inline int nvme_resv_acquire(nvme_link_t l, __u32 nsid, enum nvme_resv_racqa racqa,
 				    bool iekey, bool disnsrs, enum nvme_resv_rtype rtype,
-				    __u16 prkey, __u64 crkey, __u64 nrkey,
-				    __u32 *result)
-
+				    __u8 crkey, __u8 prkey, __u32 *result)
 {
-	__le64 payload[2] = {
-		htole64(crkey),
-		htole64(nrkey)
+	__u8 payload[2] = {
+		crkey,
+		prkey
```

Comment:
The payload is correct, no need to change.
<img width="1200" height="268" alt="image" src="https://github.com/user-attachments/assets/38c73a28-cb00-41e1-8ee3-2f578223828d" />
Thus changing the type from __u64 to __u8 is wrong IMO.